content/docs/Configuration/_index.md

```diff
@@ -722,6 +722,45 @@ jobs:
 
 This config would update Fedora Rawhide and Fedora 35 dist-git branches.
 
+##### pull_from_upstream
+
+A dist-git only job that opens a new dist-git pull request in
+src.fedoraproject.org when a new upstream release happens using a notification
+from [release-monitoring.org](https://release-monitoring.org/).
+
+This job utilizes the same logic as `propose_downstream` with the only
+exception it is defined and executed in dist-git.
+
+Use [`issue_repository`](#issue_repository) option to get information about
+possible failures during the update process.
+
+Requirements:
+* The job is defined in dist-git.
```

Comment:
What if I had it configured differently (`dist_git_branches`) for each branch? Which config/branch would be the source of truth then?

Reply:
@lbarcziova could you please answer and suggest how to capture this use case?

Reply:
We take the config from the default branch of the dist-git repository. I think for this kind of use case user could configure multiple jobs :thinking: